demosplan/DemosPlanCoreBundle/Resources/client/scss/utility/_font-size.scss

```
@@ -11,62 +11,62 @@
 
 // 50px
 .font-size-brand-hero {
```

To be in Line with the other utility classes (dropping the "domain" part of their names), i'd suggest to change it like:

```suggestion
.font-size-hero {
```